_layouts/tag-archive.html

```diff
@@ -2,12 +2,33 @@
 layout: base
 ---
 
-<h1>Posts with {{ page.type }} '{{ page.title }}'</h1>
+<div class="grid-wrapper">
+  <div class="grid__item width-10-12">
+    <h1 class="title text-caps">Tagged posts: '{{ page.title }}'</h1>
+  </div>
+  <div class="grid__item width-2-12 align-self-center">
+    <a href="{{site.baseurl}}/feed.xml"><i class="fas fa-rss-square"></i></a>
```

Comment:
if anyone knows why this square does not get orange like when used on other pages let me know :)